interop/xds/custom_lb.go

```diff
@@ -0,0 +1,161 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package xds contains a custom load balancing policy implementation for
+// interop tests.
```

Comment:
That's all it holds now, but it's called `interop/xds` so it seems like a place where we'd put some other common xDS interop code.  See other comment just below.

Reply:
Switched comment to "Package xds contains various xds interop helpers for usage in interop tests".